File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(std::vector<int>& nums) {
        int n = nums.size();
        int sum1 = (n * (n + 1) )/ 2;
        int sum2 = 0;
        for (const auto& el : nums) {
            sum2 += el;
        }
        return abs(sum1 - sum2);
    }
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = len(nums)
        full_sum = (n * (n + 1)) // 2
        element_sum = 0
        for el in nums:
            element_sum += el
        return full_sum - element_sum